src/components/Book.css

```diff
@@ -0,0 +1,106 @@
+.book-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  background-color: #fff;
+  max-width: 95rem;
+  margin: 4rem 6rem;
+  padding: 3rem;
+  box-shadow: rgba(0, 0, 0, 0.02) 0 1px 3px 0, rgba(27, 31, 35, 0.15) 0 0 0 1px;
+}
+
+.book-intial--info {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+  text-transform: capitalize;
+}
+
+.title-author {
+  line-height: 0;
+}
+
+.title {
+  font-size: 2.8rem;
+  font-weight: bold;
+}
+
+.author {
+  font-size: 1.4rem;
+  font-weight: 300;
+  color: #4386bf;
+}
```

Comment:
- [ ] Please kindly ensure that your design matches the original design guidelines. See below images for reference.
- [ ] The progress bar should be a real progress bar like in the original design
- [ ] There should be a separation bar between the action buttons(comment, remove, edit)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/232202386-fc4a0988-af57-4460-91e8-691f0422e639.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/232202480-fb793ab5-6ba0-4056-9a95-dc1217d3ce5a.png)